setup.py

```diff
@@ -75,6 +75,7 @@ def run_tests(self):
         'redis',
         'six',
         'pillow',
+        'ordereddict',
```

Comment:
`ordereddict` is needed by `flask_restful` and is missing from python 2.6 that's the reason why it is failing [more details](https://travis-ci.org/inveniosoftware/flask-iiif/jobs/44354896#L726).